server-plugins/profile.js

```diff
@@ -634,28 +703,28 @@ exports.commands = {
 				profile += `${background(toId(username))} ${showBadges(toId(username))}`;
 				profile += `<img src="${avatar}" height="80" width="80" align="left">`;
 				profile += `&nbsp;<font color="#24678d"><strong>Name:</strong></font> ${Server.nameColor(username, true)}&nbsp; ${getFlag(toId(username))} ${showTitle(username)}<br />`;
```

Comment:
We could also set the color where it says:
`<font color="24678d">`
But I'll probably do that so don't worry I guess LOL

Reply:
@DeathlyPlays I didn't intended to change the color of Name:, Group:, Bucks: etc. I just wanted to change text color of it's content like amount of bucks etc. anyway i changed it and tested.

Reply:
No worries. It was just an idea I might add on later